src/rendering/RenderUtil.cc

```diff
@@ -320,6 +320,7 @@ void RenderUtil::Update()
     this->dataPtr->scene->SetBackgroundColor(scene.Background());
     if (scene.Grid() && !this->dataPtr->enableSensors)
       this->ShowGrid();
+    this->ShowOriginAxes();
```

Comment:
Now that the plugin displays axes for entities, I think it doesn't make sense to initialize it here.

If we want a plugin that displays the world origin axes visual, here we should check `if (scene.OriginVisual() && !this->dataPtr->enableSensors)`.